### DISCUSSION TITLE
sanoid send raw. Target wants to mount pool.

### DISCUSSION THREAD

**Discussion by ckruijntjens** [jimsalterjrs/sanoid#974] [General]
Hi All,

I have a quiestion. I want to syncoid my rpool to a backupmachine. The sync runs but when it is done it mounts all backup folders over my active system folders rpool.

With this the problem is that if a new kernel is build the zpool cache file is not correct anymore. Is there a way to send a rpool recursive and recieve it on the other siede without the mount options(automount)

**Comment by paulverket** [comment]
This has happened to me as well using syncoid between two local pools. Looking at the --debug output, one can see that "canmount=on" is set for some filesystems. This happens on both v2.2.0 and the master branch from 2/11/25. I can't override it with --recvoptions. What am I missing?
